src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -25,6 +25,7 @@
   "checkInExperienceLorotaDeletionEnabled": "check_in_experience_lorota_deletion_enabled",
   "checkInExperiencePreCheckInActionLinkTopPlacement": "check_in_experience_pre_check_in_action_link_top_placement",
   "checkInExperienceTravelReimbursement": "check_in_experience_travel_reimbursement",
+  "checkInExperienceTravelLogic": "check_in_experience_travel_logic",
   "checkInExperience45MinuteReminder": "check_in_experience_45_minute_reminder",
```

Comment:
I'm wondering why this entry and the one on line 236 are added, since they don't seem to be used.

Reply:
yeah very strange.  Maybe those flags were removed after I'd branched?  In any case, I resolve this issue.